conformant = "Unsupported"
output = """
File "generics_defaults_referential.py", line 14, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 15, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 16, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 17, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 20, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 23, in <module>: class slice is not indexable [not-indexable]
File "generics_defaults_referential.py", line 24, in <module>: class slice is not indexable [not-indexable]
File "generics_defaults_referential.py", line 25, in <module>: class slice is not indexable [not-indexable]
File "generics_defaults_referential.py", line 26, in <module>: class slice is not indexable [not-indexable]
File "generics_defaults_referential.py", line 28, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 31, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 35, in <module>: class Foo is not indexable [not-indexable]
File "generics_defaults_referential.py", line 36, in <module>: class Foo is not indexable [not-indexable]
File "generics_defaults_referential.py", line 39, in <module>: class Foo is not indexable [not-indexable]
File "generics_defaults_referential.py", line 47, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 50, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 53, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 54, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 57, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 64, in Foo3: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 70, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 71, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 72, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 78, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 83, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 84, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 95, in <module>: wrong arguments [invalid-typevar]
File "generics_defaults_referential.py", line 98, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_defaults_referential.py", line 102, in <module>: Invalid type annotation 'Bar[Any, list]'  [invalid-annotation]
File "generics_defaults_referential.py", line 103, in <module>: Invalid type annotation 'Bar[int, List[int]]'  [invalid-annotation]
File "generics_defaults_referential.py", line 104, in <module>: Invalid type annotation 'Bar[int, List[int]]'  [invalid-annotation]
File "generics_defaults_referential.py", line 105, in <module>: Invalid type annotation 'Bar[int, List[str]]'  [invalid-annotation]
File "generics_defaults_referential.py", line 106, in <module>: Invalid type annotation 'Bar[int, str]'  [invalid-annotation]
"""
conformance_automated = "Fail"
errors_diff = """
Line 38: Expected 1 errors
Line 41: Expected 1 errors
Line 80: Expected 1 errors
Line 86: Expected 1 errors
Line 14: Unexpected errors ['File "generics_defaults_referential.py", line 14, in <module>: wrong arguments [invalid-typevar]']
Line 15: Unexpected errors ['File "generics_defaults_referential.py", line 15, in <module>: wrong arguments [invalid-typevar]']
Line 16: Unexpected errors ['File "generics_defaults_referential.py", line 16, in <module>: wrong arguments [invalid-typevar]']
Line 17: Unexpected errors ['File "generics_defaults_referential.py", line 17, in <module>: wrong arguments [invalid-typevar]']
Line 20: Unexpected errors ['File "generics_defaults_referential.py", line 20, in <module>: Invalid type annotation \\'Generic\\'  [invalid-annotation]']
Line 23: Unexpected errors ['File "generics_defaults_referential.py", line 23, in <module>: class slice is not indexable [not-indexable]']
Line 24: Unexpected errors ['File "generics_defaults_referential.py", line 24, in <module>: class slice is not indexable [not-indexable]']
Line 25: Unexpected errors ['File "generics_defaults_referential.py", line 25, in <module>: class slice is not indexable [not-indexable]']
Line 26: Unexpected errors ['File "generics_defaults_referential.py", line 26, in <module>: class slice is not indexable [not-indexable]']
Line 28: Unexpected errors ['File "generics_defaults_referential.py", line 28, in <module>: wrong arguments [invalid-typevar]']
Line 31: Unexpected errors ['File "generics_defaults_referential.py", line 31, in <module>: Invalid type annotation \\'Generic\\'  [invalid-annotation]']
Line 35: Unexpected errors ['File "generics_defaults_referential.py", line 35, in <module>: class Foo is not indexable [not-indexable]']
Line 36: Unexpected errors ['File "generics_defaults_referential.py", line 36, in <module>: class Foo is not indexable [not-indexable]']
Line 39: Unexpected errors ['File "generics_defaults_referential.py", line 39, in <module>: class Foo is not indexable [not-indexable]']
Line 47: Unexpected errors ['File "generics_defaults_referential.py", line 47, in <module>: wrong arguments [invalid-typevar]']
Line 50: Unexpected errors ['File "generics_defaults_referential.py", line 50, in <module>: Invalid type annotation \\'Generic\\'  [invalid-annotation]']
Line 53: Unexpected errors ['File "generics_defaults_referential.py", line 53, in <module>: wrong arguments [invalid-typevar]']
Line 54: Unexpected errors ['File "generics_defaults_referential.py", line 54, in <module>: wrong arguments [invalid-typevar]']
Line 70: Unexpected errors ['File "generics_defaults_referential.py", line 70, in <module>: wrong arguments [invalid-typevar]']
Line 71: Unexpected errors ['File "generics_defaults_referential.py", line 71, in <module>: wrong arguments [invalid-typevar]']
Line 78: Unexpected errors ['File "generics_defaults_referential.py", line 78, in <module>: wrong arguments [invalid-typevar]']
Line 83: Unexpected errors ['File "generics_defaults_referential.py", line 83, in <module>: wrong arguments [invalid-typevar]']
Line 84: Unexpected errors ['File "generics_defaults_referential.py", line 84, in <module>: wrong arguments [invalid-typevar]']
Line 95: Unexpected errors ['File "generics_defaults_referential.py", line 95, in <module>: wrong arguments [invalid-typevar]']
Line 98: Unexpected errors ['File "generics_defaults_referential.py", line 98, in <module>: Invalid type annotation \\'Generic\\'  [invalid-annotation]']
Line 102: Unexpected errors ['File "generics_defaults_referential.py", line 102, in <module>: Invalid type annotation \\'Bar[Any, list]\\'  [invalid-annotation]']
Line 103: Unexpected errors ['File "generics_defaults_referential.py", line 103, in <module>: Invalid type annotation \\'Bar[int, List[int]]\\'  [invalid-annotation]']
Line 104: Unexpected errors ['File "generics_defaults_referential.py", line 104, in <module>: Invalid type annotation \\'Bar[int, List[int]]\\'  [invalid-annotation]']
Line 105: Unexpected errors ['File "generics_defaults_referential.py", line 105, in <module>: Invalid type annotation \\'Bar[int, List[str]]\\'  [invalid-annotation]']
Line 106: Unexpected errors ['File "generics_defaults_referential.py", line 106, in <module>: Invalid type annotation \\'Bar[int, str]\\'  [invalid-annotation]']
"""
